.changeset/hot-oranges-join.md

```diff
@@ -0,0 +1,16 @@
+---
+"@logto/experience-legacy": minor
+"@logto/integration-tests": minor
+"@logto/experience": minor
+"@logto/console": minor
+"@logto/phrases": minor
+"@logto/schemas": minor
+"@logto/core": minor
+---
+
+new MFA prompt policy
+
+You can now cutomize the MFA prompt policy in the Console:
+1. Do not ask users to set up MFA - new
+2. Ask users to set up MFA during registration (skippable, one-time prompt) - new
+3. Ask users to set up MFA on their sign-in after registration (skippable, one-time prompt) - the old policy
```

Comment:
Better clarify the latest MFA policy value definitions here. 
It rakes some effort to understand what is the relation between `Mandatory` and those newly added `PromptAtSignInAndSignUp`.  
 At the first glance prompt policy and mandatory policy are two different concept.